seismic_site/seismic_site/settings.py

```diff
@@ -91,8 +91,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'risc_db',
-        'USER': 'seism',
-        'PASSWORD': 'seism',
+        'USER': 'postgres',
+        'PASSWORD': 'postgres',
```

Comment:
Made a change in the README (see PR #35) and the command for setting up the Postgres DB uses:
`POSTGRES_USER=seism -e POSTGRES_PASSWORD=seism`
So these changes aren't necessary anymore